pkg/storaged/pages.jsx

```diff
@@ -304,6 +305,35 @@
         loc.go(location);
 }
 
+function need_new_page_alert(cur_page, location) {
+    let p = get_page_from_location(location);
+    while (p) {
+        if (p == cur_page) {
+            return false;
+        }
+        p = p.parent;
+    }
+    return true;
+}
+
+export function cleanup_new_page_alerts(cur_location) {
+    const cur_page = get_page_from_location(cur_location);
+    global_alerts.filter_alerts(a => {
+        return !a.location || need_new_page_alert(cur_page, a.location);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20061-20240219-162228-2dbcaaa3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)